src/client/common/cancellation.ts

```diff
@@ -14,6 +14,7 @@ export class CancellationError extends Error {
         super(localize.Common.canceled());
     }
 }
+
```

Comment:
Revert just to not have unneeded history touch.

Reply:
Linter won't let me ;(

Reply:
Two lines aren't allowed